src/ice_shelf/MOM_ice_shelf_initialize.F90

```diff
@@ -461,8 +453,8 @@ subroutine initialize_ice_flow_from_file(bed_elev,u_shelf, v_shelf,float_cond,&
 
   floatfr_varname = "float_frac"
 
- call MOM_read_data(filename, trim(ushelf_varname), u_shelf, G%Domain, position=CORNER,scale=1.0)
- call MOM_read_data(filename,trim(vshelf_varname), v_shelf, G%Domain, position=CORNER,scale=1.0)
+! call MOM_read_data(filename, trim(ushelf_varname), u_shelf, G%Domain, position=CORNER,scale=1.0)
```

Comment:
Please explain why these two lines are commented out, but the (now unused) variables specifying the names of these variables are still being read.
Perhaps instead of commenting these lines  out, it would make more sense to test whether there is a non-zero length variable name, and only do the call to MOM_read_data() if there is a valid string for the variable name.

Reply:
These lines are uncommented.